src/App.jsx

```diff
@@ -14,33 +47,77 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [sortField, setSortField] = useState('');
+  const [reverse, setReverse] = useState('');
+  const visibleGoods = getPrepareGoods(goodsFromServer, { sortField, reverse });
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  const fnReset = () => {
```

Comment:
function name should start with a verb
```suggestion
  const reset = () => {
```